deploy/addons/aliyun_mirror.json

```diff
@@ -8,6 +8,7 @@
   "docker.io/upmcenterprises/registry-creds": "registry.cn-hangzhou.aliyuncs.com/google_containers/registry-creds",
   "quay.io/rhdevelopers/core-dns-patcher": "registry.cn-hangzhou.aliyuncs.com/google_containers/core-dns-patcher",
   "docker.io/nvidia/k8s-device-plugin": "registry.cn-hangzhou.aliyuncs.com/google_containers/k8s-device-plugin",
+  "docker.io/rocm/k8s-device-plugin": "registry.cn-hangzhou.aliyuncs.com/google_containers/rocm-k8s-device-plugin",
```

Comment:
Wasnt sure how to name this mirror image.  put something that seemed appropriate.

Reply:
I guess this is the first time that we have a duplicate image name, I'm not sure if this is going to work... Honestly it might just be better to remove the entry altogether, I'm not sure if it's going to overwrite the Nvidia one. We use a third party package to do the copying, I'll have to dig into it to see how it works. We can leave it for now and I'll try to have a more concrete answer for you on this next week.